js-challenge/weather-api/index.js

```diff
@@ -0,0 +1,56 @@
+
+function areaWhether() {
+    //プルダウンの情報を受け取る
+    var city = document.getElementById("city").value;
+    var text = '';
+    if (city === 'London') {
+        currentWhether();
+    }
+    else if (city === 'Tokyo') {
+        currentWhether();
+    }
+    else if (city === 'Paris') {
+        currentWhether();
+    }
+    else if (city === 'New York') {
+        currentWhether();
+    }
+    else if (city === 'Indonesia') {
+        currentWhether();
+    }
+};
+
+//Ajaxで情報を取得
+function currentWhether() {
+
+    var city = document.getElementById("city").value;
```

Comment:
varはやめてconstかletを使いましょう。
再代入しない変数はconstで宣言すべきです。

(重要)
このファイルの全てのvarについてconstに置き換えられないか検討してください。

```suggestion
    const city = document.getElementById("city").value;
```